src/components/application_manager/test/policy_handler_test.cc

```diff
@@ -2029,9 +2042,9 @@ TEST_F(PolicyHandlerTest, GetAppIdForSending_ExpectReturnAnyAppInNone) {
 
   test_app.insert(mock_app_in_none_1);
 
-  std::shared_ptr<application_manager_test::MockApplication>
-      mock_app_in_none_2 =
-          std::make_shared<application_manager_test::MockApplication>();
+  std::shared_ptr<
+      NiceMock<application_manager_test::MockApplication> > mock_app_in_none_2 =
+      std::make_shared<NiceMock<application_manager_test::MockApplication> >();
```

Comment:
@ValeriiMalkov Don't you think we can use typedef for `NiceMock<application_manager_test::MockApplication>` ?